src/Features/Navigation/CoreMenu.php

```diff
@@ -57,7 +57,7 @@ public static function get_setting_items() {
 					'parent'     => 'settings',
 					'title'      => $setting,
 					'capability' => 'manage_woocommerce',
-					'id'         => $key,
+					'id'         => 'settings/' . $key,
```

Comment:
Curious why we using `settings/` here but `woocommerce/` as the prefix below?

I would expect `wc-settings/` and `wc-status/` or maybe `settings/` and `tools/`.

Does this also mean we will be keeping IDs unique and not concatenating IDs with parent IDs?